4-WebApp-your-API/4-1-MyOrg/README-use-keyvault-certificate.md

```diff
@@ -0,0 +1,77 @@
+# Web app using client certificates with Microsoft.Identity.Web
```

Comment:
>Web app using client certificates with Microsoft.Identity.Web [](http://example.com/codeflow?start=2&length=61)

How to use certificates instead of secrets in your client applications